hack/setup-dev.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+root=$(dirname "${BASH_SOURCE[0]}")
+
+bash -e $root/install-dev-deps.sh
```

Comment:
as discussed before, i'd remove this and replace perhaps with a check to make sure all the required tools are present